### PR TITLE
chore: replace custom navigation bar by UINavigationController [WPB-7307]

### PR DESCRIPTION
**Pull request by caldrian**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-7307" title="WPB-7307" target="_blank">StoryWPB-7307</a>  [iOS] : Remove "Contacts" "Folder"
  </td></table>
  <br />
 

Updates the epic branch in order to pull in the latest changes from `develop`, where the custom navigation bar is already removed.

---

### Checklist

- [x] Title contains a reference JIRA issue number like `[WPB-XXX]`.
- [x] Description is filled and free of optional paragraphs.
- [x] Adds/updates automated tests.

### CODE REVIEW

**Review by typfel — APPROVED** [review]

File: wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ConversationListViewController.swift [hunk 1]
```diff
@@ -97,7 +96,7 @@ final class ConversationListViewController: UIViewController {
         self.selfProfileViewControllerBuilder = selfProfileViewControllerBuilder
 
         let bottomInset = ConversationListViewController.contentControllerBottomInset
-        listContentController = ConversationListContentController(userSession: viewModel.userSession)
+        listContentController = .init(userSession: viewModel.userSession)
```
Comment (agisilaos) [inline wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ConversationListViewController.swift:99]: For me the use of .init makes the code less readable and harder to understand at a glance, especially when identifying which controller or object is being initiated. Using the full name of the object would improve clarity and comprehension.

File: wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/ListContent/ConversationListContentController/ConversationListContentController.swift [hunk 2]
```diff
@@ -48,7 +48,7 @@ final class ConversationListContentController: UICollectionViewController, Popov
         flowLayout.minimumLineSpacing = 0
         flowLayout.minimumInteritemSpacing = 0
         flowLayout.sectionInset = .zero
-        self.listViewModel = ConversationListViewModel(userSession: userSession)
+        listViewModel = .init(userSession: userSession)
```
Comment (agisilaos) [inline wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/ListContent/ConversationListContentController/ConversationListContentController.swift:51]: Same here.

**Review by agisilaos — APPROVED** [review]
Left two comments!

File: wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ConversationListViewController.swift (hunk 1 above)
Comment (caldrian) [reply wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ConversationListViewController.swift:99]: I'm not sure I agree on this, but let's have a separate discussion eventually, please.

File: wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/ListContent/ConversationListContentController/ConversationListContentController.swift (hunk 2 above)
Comment (caldrian) [reply wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/ListContent/ConversationListContentController/ConversationListContentController.swift:51]: See my comment https://github.com/wireapp/wire-ios/pull/1465#discussion_r1609498315

**Pull request merged**